CONTRIBUTING.md

```diff
@@ -283,10 +287,10 @@ Then, update `executors.docker_executor.docker[0].image` in [.circleci/config.ym
 
 We follow [Semantic versioning](https://semver.org/) for the plugin releases:
 
-* MAJOR version must be bumped for each plugin release that stops supporting any IDEA build (typically when `sinceBuild` is increased). <br/>
```

Comment:
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/1612 for further updates here